aws/src/main/java/org/apache/iceberg/aws/s3/S3FileIO.java

```diff
@@ -275,6 +275,7 @@ private List<String> deleteBatch(String bucket, Collection<String> keysToDelete)
         DeleteObjectsRequest.builder()
             .bucket(bucket)
             .delete(Delete.builder().objects(objectIds).build())
+            .overrideConfiguration(S3RequestUtil.disableStrongIntegrityChecksums())
```

Comment:
Sorry late for this, just saw the devlist. Have we discussed if we could introduce a config for this (can default to false), so people can still turn this check on if they want to?

Reply:
We already have `s3.checksum-enabled` in `S3FileIOProperties`. Should we reuse it, or would it be better to introduce a new one?

Reply:
I think they're slightly different checksums; there's always been md5 in places (bulk delete?) ; AWS S3 and this SDK have moved to a stronger checksum algorithm. When that is disabled you still need to restore the MD5 signing where retired.

It may be best to move to an enum here, e.g. "none, classic, strong" to cope with any future changes.